shimming-toolbox/shimmingtoolbox/cli/maths.py

```diff
@@ -77,44 +88,63 @@ def phase(fname_im, fname_real, fname_output, verbose):
     create_output_dir(fname_output, is_file=True)
 
     # Load input
-    nii_real = nib.load(fname_real)
-    nii_im = nib.load(fname_im)
-
-    # Calculate the phase
-    phase = np.arctan2(nii_im.get_fdata(), nii_real.get_fdata())
+    if fname_complex is not None:
+        nii = nib.load(fname_complex)
+        # Calculate the phase
+        phase_data = np.angle(np.asanyarray(nii.dataobj))
+        fname_json = splitext(fname_complex)[0] + '.json'
+    elif fname_real is not None and fname_im is not None:
+        nii = nib.load(fname_real)
+        nii_im = nib.load(fname_im)
+        # Calculate the phase
+        phase_data = np.arctan2(nii_im.get_fdata(), nii.get_fdata())
+        fname_json = splitext(fname_real)[0] + '.json'
+    else:
+        raise ValueError("At least one of the inputs must be provided.")
```

Comment:
The error message should be changed. Right now, if someone submits the real data only, an error message saying "at least one input" would show up while one input was given

Reply:
The error message will never show up since I added some "cloup" constraints on the command line inputs. The euro message is still wrong so I modified it anyways [d65e771](https://github.com/shimming-toolbox/shimming-toolbox/pull/583/commits/d65e771884a8d29f1045826d65ccf6c919f24296)